### PR TITLE
[ADF-4166] move app-list-cloud.page to adf-testing

### PR DESCRIPTION
**Pull request by gmandakini**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**
moved UserInfoDialogPage, AppListCloudPage, LoginSSOPage and TabsPage under adf-testing.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-4166

### CODE REVIEW

File: e2e/pages/adf/process-services/taskDetailsPage.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@
 import { AppSettingsToggles } from './dialog/appSettingsToggles';
 import { Util } from '../../../util/util';
 import { element, by, protractor, browser } from 'protractor';
-import { TabsPage } from '../material/tabsPage';
+import { TabsPage } from '../../../../lib/testing/src/lib/material/tabsPage';
```
Comment (mauriziovitale) [inline e2e/pages/adf/process-services/taskDetailsPage.ts]: wrong path

File: e2e/pages/adf/viewerPage.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 import { Util } from '../../util/util';
 
-import { TabsPage } from './material/tabsPage';
+import { TabsPage } from '../../../lib/testing/src/lib/material/tabsPage';
```
Comment (mauriziovitale) [inline e2e/pages/adf/viewerPage.ts]: wrong path

File: e2e/pages/adf/commentsPage.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@
 import { element, by } from 'protractor';
 
 import { Util } from '../../util/util';
-import { TabsPage } from './material/tabsPage';
+import { TabsPage } from '../../../lib/testing/src/lib/material/tabsPage';
```
Comment (cristinaj) [inline e2e/pages/adf/commentsPage.ts]: Shouldn't this import be something like: import { TabsPage } from '@alfresco/adf-testing';

File: e2e/core/user-info-component-cloud.e2e.ts [hunk 4]
```diff
@@ -15,20 +15,20 @@
...
+import { LoginSSOPage } from '@alfresco/adf-testing';
 import { SettingsPage } from '../pages/adf/settingsPage';
 import TestConfig = require('../test.config');
 import { browser } from 'protractor';
 import { NavigationBarPage } from '../pages/adf/navigationBarPage';
-import { UserInfoDialog } from '../pages/adf/dialog/userInfoDialog';
+import { UserInfoDialogPage } from '@alfresco/adf-testing';
 import { Identity } from '../actions/APS-cloud/identity';
 
 describe('User Info - SSO', () => {
```
Comment (mauriziovitale) [inline e2e/core/user-info-component-cloud.e2e.ts]: This e2e file should be moved under the /e2e/process-cloud folder

File: lib/testing/src/lib/core/pages/user-info-dialog.page.ts [hunk 5]
```diff
@@ -15,11 +15,11 @@
...
  */
 
-import { Util } from '../../../util/util';
-import { TabsPage } from '../material/tabsPage';
 import { element, by, browser, protractor } from 'protractor';
+import { BrowserVisibility } from '../browser-visibility';
+import { TabsPage } from '../../material/tabsPage';
 
-export class UserInfoDialog {
+export class UserInfoDialogPage {
```
Comment (cristinaj) [inline lib/testing/src/lib/core/pages/user-info-dialog.page.ts]: Can we rename this file to UserInfoPage? It's not only about the dialog.

File: lib/testing/src/lib/core/pages/user-info-dialog.page.ts [hunk 6]
```diff
@@ -15,11 +15,11 @@
  * limitations under the License.
  */
 
-import { Util } from '../../../util/util';
-import { TabsPage } from '../material/tabsPage';
 import { element, by, browser, protractor } from 'protractor';
+import { BrowserVisibility } from '../browser-visibility';
+import { TabsPage } from '../../material/tabsPage';
```
Comment (mauriziovitale) [inline lib/testing/src/lib/core/pages/user-info-dialog.page.ts]: wrong path

File: e2e/core/user-info-component-cloud.e2e.ts (hunk 4 above)
Comment (gmandakini) [reply e2e/core/user-info-component-cloud.e2e.ts]: The userInfoDialog has got both Content and Process elements in it. Hence I have put it under core, for now.

File: lib/testing/src/lib/core/pages/user-info-dialog.page.ts (hunk 5 above)
Comment (gmandakini) [reply lib/testing/src/lib/core/pages/user-info-dialog.page.ts]: done

File: lib/testing/src/lib/core/pages/user-info-dialog.page.ts (hunk 6 above)
Comment (gmandakini) [reply lib/testing/src/lib/core/pages/user-info-dialog.page.ts]: this file is in the adf-testing as well, hence using a relative path.

File: e2e/pages/adf/viewerPage.ts (hunk 2 above)
Comment (gmandakini) [reply e2e/pages/adf/viewerPage.ts]: done

File: e2e/pages/adf/process-services/taskDetailsPage.ts (hunk 1 above)
Comment (gmandakini) [reply e2e/pages/adf/process-services/taskDetailsPage.ts]: done

File: e2e/pages/adf/commentsPage.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (gmandakini) [reply e2e/pages/adf/commentsPage.ts]: done

File: lib/testing/src/lib/core/pages/user-info.page.ts [hunk 7]
```diff
@@ -37,17 +37,23 @@ export class UserInfoDialog {
...
         return this;
     }
 
+    clickUserProfile() {
+        BrowserVisibility.waitUntilElementIsVisible(this.userProfileButton);
+        this.userProfileButton.click();
+    }
+
     clickOnContentServicesTab() {
         let tabsPage = new TabsPage;
```
Comment (cristinaj) [inline lib/testing/src/lib/core/pages/user-info.page.ts]: the instantiation needs to be done like this: new TabsPage();

**Pull request merged**